### ISSUE TITLE
v0.1.44 doesn't support symlinks anymore

### ISSUE THREAD

**Issue by kapalla** [martinlindhe/laravel-vue-i18n-generator#108]
#105 changes the behavior how folders are getting registered in an array.

1. **Directory structure**
packages/core/src/resources/lang/de/help.php
packages/core/src/resources/lang/en/help.php
packages/user/src/resources/lang/de/form.php
packages/user/src/resources/lang/en/form.php

2. **Symlinks**
resources/lang/vendor/core -> packages/core/src/resources/lang
resources/lang/vendor/user -> packages/user/src/resources/lang

`php artisan vue-i18n:generate --with-vendor --ansi`

shows the following error message

``` 
  1   DirectoryIterator::__construct("/Users/kai/Sites/my-project/resources/lang/vendor/lang")
      /Users/kai/Sites/my-project/vendor/martinlindhe/laravel-vue-i18n-generator/src/Generator.php:218

  2   MartinLindhe\VueInternationalizationGenerator\Generator::allocateLocaleArray("/Users/kai/Sites/my-project/resources/lang/vendor/lang")
      /Users/kai/Sites/my-project/vendor/martinlindhe/laravel-vue-i18n-generator/src/Generator.php:236

  3   MartinLindhe\VueInternationalizationGenerator\Generator::allocateLocaleArray("/Users/kai/Sites/my-project/resources/lang/vendor")
      /Users/kai/Sites/my-project/vendor/martinlindhe/laravel-vue-i18n-generator/src/Generator.php:81

  4   MartinLindhe\VueInternationalizationGenerator\Generator::generateFromPath("/Users/kai/Sites/my-project/resources/lang", "es6")
      /Users/kai/Sites/my-project/vendor/martinlindhe/laravel-vue-i18n-generator/src/Commands/GenerateInclude.php:67

  …
```

**Comment by martinlindhe** [comment]
@kapalla Alright, thanks. We'll revert that PR for now and push another release

**Comment by martinlindhe** [comment]
Patch reverted and released 0.1.45

**Comment by kapalla** [comment]
I honestly don't understand the problem of #108

`(new SplFileInfo($fileinfo))->getFilename()` resolves now the symlink and returns **lang** instead of **core** and **user** when calling `$fileinfo->getFilename()` directly.
